en/sim_gazebo_gz/README.md

```diff
@@ -31,16 +31,42 @@ sudo apt-get install gz-garden
 
 ## Running the Simulation
 
-Gazebo SITL simulation can be conveniently run through a `make` command as shown below:
+Gazebo SITL simulation can be conveniently run using a `make` command as shown below:
 
 ```sh
 cd /path/to/PX4-Autopilot
 make px4_sitl gz_x500
 ```
 
-This will run both the PX4 SITL instance and the Gazebo client.
+This runs both the PX4 SITL instance and the Gazebo client.
 Note that all gazebo make targets have the prefix `gz_`.
 
+Another way that Gazebo SITL can be connected is in _standalone mode_.
```

Comment:
Why would I ever use standalone mode? As a reader this looks a lot harder than just doing a make command as above.

Reply:
Because you can launch Gazebo and PX4 across any two devices on your network and they will still connect. Up to now, PX4 and Gazebo always had to be on the same host. Now you can launch PX4 in a docker container or on a different PC across the building and Gazebo will still be able to connect. Furthermore, if you use a VPN, you can even connect the two from different networks.

Reply:
This is worth saying. Done.